### ISSUE TITLE
kubectl errors polluting the output with "proto: duplicate proto type registered: google.protobuf.Any"

### ISSUE THREAD

**Issue by zoobab** [Mirantis/kubeadm-dind-cluster#76]
Hi,

I am using dind-cluster-v1.7.sh, and when I do a docker exec -it 1ab4e5f7 /bin/bash, and run "kubectl get pods", the output is polluted by this:

```
root@kube-master:/# kubectl get pods
2018-02-06 13:29:42.213277 I | proto: duplicate proto type registered: google.protobuf.Any
2018-02-06 13:29:42.214510 I | proto: duplicate proto type registered: google.protobuf.Duration
2018-02-06 13:29:42.214661 I | proto: duplicate proto type registered: google.protobuf.Timestamp
NAME                                  READY     STATUS             RESTARTS   AGE
busybox-envvars                       1/1       Running            0          2h
busybox-ping-googleyahoo-onepod       2/2       Running            0          2h
busybox-ping-localhost                1/1       Running            0          2h
```

Any idea why? Maybe the client and the server mismatch?

**Comment by zoobab** [comment]
See also https://github.com/kubernetes/kubectl/issues/30

**Comment by lukaszo** [comment]
According to issue which you linked it's fixed in k8s 1.8. I didn't see it in 1.8 and 1.9 so closing for now. Feel free to reopen if you still see it when using newer clusters.
